src/main/java/org/jalasoft/moi/model/Laguage.java

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright (c) 2020 Jalasoft.
+ *
+ * This software is the confidential and proprietary information of Jalasoft.
+ * ("Confidential Information"). You shall not disclose such Confidential
+ * Information and shall use it only in accordance with the terms of the
+ *  license agreement you entered into with Jalasoft.
+ */
+
+package org.jalasoft.moi.model;
+
```

Comment:
I think you need comment this enum and explain what is for

Reply:
I was think so, but I'm not sure what should be in, Could you give me some ideas?